Tasks/AzureRmWebAppDeploymentV4/task.json

```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 4,
         "Minor": 205,
```

Comment:
This should be 206 and the Patch should be 0. (assuming we will manage to merge it this week)

Reply:
Hi @mmrazik - Martin, this is fixed now.